docs/en/quick-answers/troubleshooting/cant-use-file-linking-toolkit-app-chrome.md

```diff
@@ -30,39 +30,33 @@ If you have launched Chrome before allowing the Shotgun Desktop to register the
 
```

Comment:
line 27: "we recommend to type" – maybe "we recommend typing" or "we recommend that you type" instead?